app/views/whitehall_migration/documents.html.erb

```diff
@@ -1,6 +1,61 @@
-<h1>Documents</h1>
+<% content_for :back_link, render_back_link(href: whitehall_migration_path(@whitehall_migration)) %>
+<% params[:state] ? title = "Document Imports: #{params[:state].titleize}" : title = "Document Imports" %>
```

Comment:
It'd be more common to write this as <% title = params[:state] ? "Document Imports: #{params[:state].titleize}" : "Document Imports" %>